src/py42/services/storage/_auth.py

```diff
@@ -36,12 +48,14 @@ def get_tmp_auth(self):
             u"destinationGuid": self._destination_guid,
         }
         response = self._connection.post(uri, json=data)
-        return response
+        storage_url = response[u"serverUrl"]
```

Comment:
moving this and lines 67-69 into a shared private method is just about the only thing I could _maybe_ suggest on this PR. It's debatable whether or not that's even the right thing to do because if one of those endpoint were to change and not the other, we'd want to have the logic separated anyway.

Reply:
Yeah, I don't see any real benefit in pulling out such a simple task into its own method.